src/main/resources/logback.xml

```diff
@@ -61,6 +61,7 @@
         <appender-ref ref="console"/>
     </logger>
     -->
+
```

Comment:
keeping format consistent with the rest of the file

Reply:
Should we delete this code since it is commented out?

Reply:
historically we've kept it commented out in all apps that use slick because we don't want it on all the time but it's still very useful for debugging slick queries